### DISCUSSION TITLE
Default Personal preset

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#24322] [Ideas]
It's slightly annoying to enable the same mod preset a few times a day. I propose to make it possible to set the default preset - one that would be enabled automaticly after game starting. It would also be nice to have such presets enabled automatically in multiplayer as soon as a combination of mods is available for selection.

**Comment by peppy** [comment]
100% planned. Implementation shouldn't be too hard (add new property in realm `IsDefault`, add context menu option to set default, use first available default on entering song select with no mods applied).
